cwl_wes/custom_config.py

```diff
@@ -111,6 +361,30 @@ class IdConfig(FOCABaseConfig):
 
 
 class CustomConfig(FOCABaseConfig):
+    """Model for custom configuration parameters.
+
+    Args:
+        storage: Storage config parameters.
+        celery: Celery config parameters.
+        endpoint_params: Endpoint config parameters.
+        service_info: Service Info config parameters.
+        tes_server: TES Server config parameters.
+        drs_server: DRS Server config parameters.
+        runs_id: Identifier config parameters.
+
+    Attributes:
+        storage: Storage config parameters.
+        celery: Celery config parameters.
+        endpoint_params: Endpoint config parameters.
+        service_info: Service Info config parameters.
+        tes_server: TES Server config parameters.
+        drs_server: DRS Server config parameters.
+        runs_id: Identifier config parameters.
+
+    Raises:
+        pydantic.ValidationError: The class was instantianted with an illegal
+            data type.
+    """
     storage: StorageConfig = StorageConfig()
     celery: CeleryConfig = CeleryConfig()
     endpoint_params: EndpointConfig = EndpointConfig()
```

Comment:
Maybe better to rename to `endpoint`, or even better, `controller`. In any case it should match the name of the class, minus `Config`.